### PR TITLE
Token regex update

### PR DESCRIPTION
**Pull request by KartulUdus**
Hello

some users of my bot started reporting that their tokens we're not accepted.
2 of them had generated tokens starting with "N" 

* updated regex to also accept tokens starting with N

### CODE REVIEW

**Comment by GiovanniMCMXCIX** [comment]
When did you create those 2 bots?

**Comment by KartulUdus** [comment]
Yesterday or the day before. Both of them also tried to regenerate a token and got another one beginning with 'N'

**Comment by Xpktro** [comment]
Hello there, I just stumbled upon this issue as well. Was about to drop disco but traced back the "Invalid token passed" message to this regex and realized my bot token starts with an N

File: disco/util/token.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import re
 
-TOKEN_RE = re.compile(r'M\w{23}\.[\w-]{6}\..{27}')
+TOKEN_RE = re.compile(r'[M-N]\w{23}\.[\w-]{6}\..{27}')
```
Comment (SpencerSharkey) [inline disco/util/token.py]: I think simply using `\w{24}\.[\w-]{6}\..{27}` provides enough validation for Discord's token format and adds more value to this update.

Who knows when the next character will pop up, and Disco shouldn't have to be updated when this inevitably occurs.

**Comment by KartulUdus** [comment]
Done. Should you choose to go with no validation instead, let me know please

**Comment by b1naryth1ef** [comment]
I've removed the token checking module entirely in master.

**Pull request closed without merging**